<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Test test1_SUITE:ms_test result</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<a name="top"></a><pre>
=== Test case: <a href="test1_suite.src.html#ms_test-1">test1_SUITE:ms_test/1</a> (click for source code)

=== Config value:

    [{watchdog,&lt;0.204.0&gt;},
     {tc_logfile,"/home/yaroslav/Desktop/Erlang/erlang/logs/ct_run.ct@yaroslav-VirtualBox.2022-08-26_17.53.27/Erlang.erlang.logs/run.2022-08-26_17.53.28/test1_suite.ms_test.html"},
     {tc_group_properties,[{suite,test1_SUITE}]},
     {tc_group_path,[]},
     {data_dir,"/home/yaroslav/Desktop/Erlang/erlang/test1_SUITE_data/"},
     {priv_dir,"/home/yaroslav/Desktop/Erlang/erlang/logs/ct_run.ct@yaroslav-VirtualBox.2022-08-26_17.53.27/Erlang.erlang.logs/run.2022-08-26_17.53.28/log_private/"}]

=== Current directory is "/home/yaroslav/Desktop/Erlang/erlang/logs/ct_run.ct@yaroslav-VirtualBox.2022-08-26_17.53.27"

=== Started at 2022-08-26 17:53:31

<br />
Master: &lt;0.245.0&gt;
Slaves: [&lt;0.246.0&gt;,&lt;0.247.0&gt;,&lt;0.248.0&gt;,&lt;0.249.0&gt;,&lt;0.250.0&gt;]

Slave 1 got message "Hi, slave!"

Slave 2 got message "Hi, slave!"

Slave 3 got message "Hi, slave!"

Slave 4 got message "Hi, slave!"

Slave 5 got message "Hi, slave!"

Master restarting dead slave 2

OMG someone died.. It was: &lt;0.247.0&gt;


</pre>
<div class="sasl"><pre><b>*** System report during test1_SUITE:ms_test/1 2022-08-26 17:53:32.732 ***</b><a id=e-1 class="link-to-entry" href="#e-1">&#x1f517;</a>
=ERROR REPORT==== 26-Aug-2022::17:53:32.732463 ===
Error in process &lt;0.245.0&gt; on node 'ct@yaroslav-VirtualBox' with exit value:
{badarith,[{mf,setnth,3,[{file,"mf.erl"},{line,8}]},
           {ms,msLoop,1,[{file,"ms.erl"},{line,38}]}]}


</pre></div>
<pre>

</pre>
<div class="ct_error_notify"><pre><b>*** CT Error Notification 2022-08-26 17:53:33.739 ***</b><a id=e-2 class="link-to-entry" href="#e-2">&#x1f517;</a>
erlang:send failed
Reason: {badarg,[{erlang,...},{...}|...]}

<a href="#end">Full error description and stacktrace</a>
</pre></div>
<pre>
<a name="end"></a>


=== Ended at 2022-08-26 17:53:33
=== Location: [{erlang,send},
 {ms,to_slave,19},
 {test1_SUITE,'-ms_test_common/1-lc$^2/1-1-',239},
 {test1_SUITE,ms_test_common,239},
 {test1_SUITE,ms_test,211},
 {test_server,ts_tc,1783},
 {test_server,run_test_case_eval1,1292},
 {test_server,run_test_case_eval,1224}]
=== === Reason: bad argument
  in function  erlang:send/2
     called as erlang:send(master,{"Hi, slave!",1})
     *** argument 1: invalid destination
  in call from ms:to_slave/2 (ms.erl, line 19)
  in call from test1_SUITE:'-ms_test_common/1-lc$^2/1-1-'/1 (test1_SUITE.erl, line 239)
  in call from test1_SUITE:ms_test_common/1 (test1_SUITE.erl, line 239)
  in call from test1_SUITE:ms_test/1 (test1_SUITE.erl, line 211)
  in call from test_server:ts_tc/3 (test_server.erl, line 1783)
  in call from test_server:run_test_case_eval1/6 (test_server.erl, line 1292)
  in call from test_server:run_test_case_eval/9 (test_server.erl, line 1224)
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2022 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Aug 26 2022 17:53:31<!--/date--><br />
</div>
</center>
</body>
</html>
